.eslintrc

```diff
@@ -164,7 +163,6 @@
 
     /* Stylistic Issues */
     "array-bracket-spacing": [1, "never"],
-    "brace-style": [1, "stroustrup", { "allowSingleLine": true }],
```

Comment:
Current style has if statements on the same line. We can add this back later when we change over to linting only in eslint.